edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/BrokerServiceIdentityTest.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test
+{
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Newtonsoft.Json;
+    using Xunit;
+    public class BrokerServiceIdentityTest
+    {
+        [Fact]
+        public void SerializeTest()
+        {
+            string id = "id";
+            string authChain = "authChain";
+            BrokerServiceIdentity brokerServiceIdentity = new BrokerServiceIdentity(id, Option.Some(authChain));
+            var s = JsonConvert.SerializeObject(brokerServiceIdentity);
+            Assert.Equal($"{{\"Identity\":\"{id}\",\"AuthChain\":\"{authChain}\"}}", s);
+        }
```

Comment:
Assuming we don't need to test serialization without an auth chain, because it shouldn't error if this passes